x/sequencer/keeper/proposer.go

```diff
@@ -12,6 +12,9 @@ import (
 
 func (k Keeper) abruptRemoveProposer(ctx sdk.Context, rollapp string) {
 	proposer := k.GetProposer(ctx, rollapp)
+	if proposer.Sentinel() {
+		return
+	}
```

Comment:
this was the bug
Since kick scenario will clear the proposer then hard fork then try to clear it again (via hook)